src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  const adder = (firstValue) => {
```

Comment:
Consider changing the parameter name 'firstValue' to something more descriptive like 'args' or 'value'